core/src/main/scala/kafka/server/ClientQuotaManager.scala

```diff
@@ -220,6 +216,14 @@ class ClientQuotaManager(private val config: ClientQuotaManagerConfig,
     ClientSensors(quotaSensor, throttleTimeSensor)
   }
 
+  private def getThrottleTimeSensorName(clientId: String): String = {
+    apiKey + "ThrottleTime-" + clientId
+  }
+
+  private def getQuotaSensorName(clientId: String): String = {
+    apiKey + "-" + clientId
+  }
+
```

Comment:
I'm not sure we are very consistent about this but I have observed it both ways. I'll clean it up to make it a single line